pinot-segment-local/src/main/java/org/apache/pinot/segment/local/upsert/TableUpsertMetadataManagerFactory.java

```diff
@@ -61,6 +64,12 @@ public static TableUpsertMetadataManager create(TableConfig tableConfig,
         upsertConfig.setEnablePreload(
             Boolean.parseBoolean(instanceUpsertConfig.getProperty(UPSERT_DEFAULT_ENABLE_PRELOAD, "false")));
       }
+
+      // server level config honoured only when table level config is not set to true
+      if (!upsertConfig.isAllowPartialUpsertConsumptionDuringCommit()) {
+        upsertConfig.setAllowPartialUpsertConsumptionDuringCommit(Boolean.parseBoolean(
+            instanceUpsertConfig.getProperty(UPSERT_DEFAULT_ALLOW_PARTIAL_UPSERT_CONSUMPTION_DURING_COMMIT, "false")));
```

Comment:
we wanted to keep this `true` by default right?

Reply:
Ah my bad! I see we have changed the behaviour from `pause` to `allow`. Then this makes sense!